.travis/run-unit-tests.sh

```diff
@@ -40,14 +40,14 @@ else
 
     if [ "${TRAVIS_PULL_REQUEST}" = "false" ] ; then
         # This is not a PR build.
-        COMMIT1="$(echo ${TRAVIS_COMMIT_RANGE} | cut -f 1 -d '.')"
-        COMMIT2="$(echo ${TRAVIS_COMMIT_RANGE} | cut -f 4 -d '.')"
+        COMMIT1="$(echo "${TRAVIS_COMMIT_RANGE}" | cut -f 1 -d '.')"
+        COMMIT2="$(echo "${TRAVIS_COMMIT_RANGE}" | cut -f 4 -d '.')"
 
-        if [ "$(git cat-file -t ${COMMIT1} 2>/dev/null)" = commit -a "$(git cat-file -t ${COMMIT2} 2>/dev/null)" = commit ] ; then
+        if [ "$(git cat-file -t "${COMMIT1}" 2>/dev/null)" = commit ] && [ "$(git cat-file -t "${COMMIT2}" 2>/dev/null)" = commit ] ; then
```

Comment:
Can we omit the space preceding ; as we do in other places?

```suggestion
        if [ "$(git cat-file -t "${COMMIT1}" 2>/dev/null)" = commit ] && [ "$(git cat-file -t "${COMMIT2}" 2>/dev/null)" = commit ]; then
```